fabric-common/types/index.d.ts

```diff
@@ -294,6 +316,7 @@ export class Client {
 	public getChannel(name: string): Channel;
 
 	public setTlsClientCertAndKey(clientCert: string, clientKey: string): Client;
+	public setTlsClientCertAndKey(): Client;
```

Comment:
Are the parameters needed here ?

Reply:
You tell me! You wrote code in Gateway that calls it either with both arguments or with none. And the code in Client.js has explicit code paths for when both argument are supplied and when they are not both supplied (which uses neither). This overload means that the Gateway code that calls it with either two or zero arguments can compile.

Reply:
Sounds good, thanks